lib/app.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express')
+const app = express()
+const logSymbols = require('log-symbols')
+const log = require('fancy-log')
+const db = require('../db')
+const addToLibreNet6 = require('../sendConfig')
```

Comment:
If there any chance that we unify the module name and the local variable, it helps to code trace-ability.

Reply:
solved in https://github.com/LibreRouterOrg/soporteremoto-tiers/pull/1/commits/a5c716296f74a5bd031b534e5bfc216d9a242b2d